keras_contrib/layers/advanced_activations.py

```diff
@@ -283,3 +283,75 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 get_custom_objects().update({'Swish': Swish})
+
+
+class ReLUs(Layer):
+    """Rectified Linear Unit with Sigmoid to generate oscilations.
```

Comment:
Can you report the performances of this activation on the datasets you mentioned (please include standard deviation)?

Reply:
Hi @tboquet,

Busy doing that now, but training the three datasets take a while. I also want to get a nice outcome for the SineRelu, so checking different epsilons here.

Will share the final loss, accuracy and standard deviation of loss in the docstrings.

Cheers,
Wilder